steps/discover-repos.rb

```diff
@@ -28,18 +28,18 @@
 require 'fileutils'
 require 'slop'
 require 'octokit'
-
-max = 1000
+require 'date'
 
 opts = Slop.parse do |o|
   o.string '--token', 'GitHub access token', default: ''
   o.boolean '--dry', 'Make no round-trips to GitHub API (for testing)', default: false
   o.integer '--total', 'Total number of repos to take from GitHub', required: true
   o.integer '--pause', 'How many seconds to sleep between API calls', default: 10
   o.integer '--page-size', 'Number of repos to fetch in one API call', default: 100
-  o.integer '--min-stars', 'Minimum GitHub stars in each repo', default: max
```

Comment:
@lueFlake what is the point of this change?

Reply:
@yegor256 why would we need to keep "max" variable? It is used only once here and i dont understand logic begind it in this line